tests/ingress_test.go

```diff
@@ -0,0 +1,170 @@
+package tests
+
+import (
+	"testing"
+
+	netv1 "k8s.io/api/networking/v1"
+	"k8s.io/utils/pointer"
+
+	"github.com/jinzhu/copier"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIngress(t *testing.T) {
+	chart := LoadChart(t)
+
+	tests := []struct {
+		Name       string
+		ValuesFunc func(v *CoderValues)
```

Comment:
nit (non-blocking): Could you add a comment describing what this does?

Reply:
added!